capacity-factor/info.yml

```diff
@@ -0,0 +1,12 @@
+---
+uid: capacity-factor
+title: Capacity Factor
+prerequisites:
+  - None
+learning_objectives:
+  - Calculate capacity factor.
+references:
+  - tsoulfanidis_nuclear_2013
```

Comment:
yes. indeed, if we're clever we should tie it to a group zotero library... but for now maybe it's better to dump the reference in full text instead of the key.